cvcl/templates/master.html

```diff
@@ -28,7 +31,7 @@
                 <span class="icon-bar"></span>
                 <span class="icon-bar"></span>
             </button>
-            <a class="navbar-brand" href="{% url 'index' %}" style="padding-top:15px;">Cloud VCL</a> <!-- Padding is necessary with material theme -->
+            <a id="navbrand" class="navbar-brand" href="{% url 'index' %}">Cloud VCL</a> <!-- Padding is necessary with material theme -->
```

Comment:
Is the navbar-brand class used anywhere else? Why not use it to identify the style?